.github/workflows/docs.yml

```diff
@@ -26,20 +30,12 @@ jobs:
       - working-directory: gir
         run: cargo build --release
       - run: cargo install rustdoc-stripper
+      - run: git clone https://gitlab.gnome.org/World/Rust/gir-rustdoc/ # checkout action doesn't support random urls
+      - run: eval $(python3 ./gir-rustdoc/gir-rustdoc.py pre-docs)
```

Comment:
I think you want to specify `--default-branch` here, because it defaults to 'main' and github does not want to tell me the default branch.